RickAndMortyFunApp/SearchViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  SearchViewController.swift
+//  RickAndMortyFunApp
+//
+//  Created by Rusłan Chamski on 24/02/2019.
+//  Copyright © 2019 Rusłan Chamski. All rights reserved.
+//
+
+import UIKit
+
+class SearchViewController: UIViewController {
```

Comment:
What is SearchViewController? Do we need it as a separate view controller? We could search from within each of other view controllers as a built-in feature